package.json

```diff
@@ -5,7 +5,8 @@
   "main": "src/prebid.js",
   "scripts": {
     "test": "gulp test",
-    "lint": "gulp lint"
+    "lint": "gulp lint",
+    "build_prebid_js": "gulp build --modules=criteoBidAdapter"
```

Comment:
this is what creates the prebid file we currently have
Add adapters to this list once you verified they work properly